glad/stages/validate.py

```diff
@@ -0,0 +1,27 @@
+from glad.utils.utils import file_details
+from osgeo import gdal
+
+
+def validate_tiles(tiles, **kwargs):
+    for tile in tiles:
+        f_name, year, folder, tile_id = file_details(tile)
+
+        if f_name == "day.tif":
+            expected_min = 0
+            expected_max = 366
+        else:
+            expected_min = 0
+            expected_max = 3
```

Comment:
I would make the first a `range(0,367)` and the second a list `[0,2,3]`, since we don't expect values of 1 here.
then check if `min`/ `max` is within.

Reply:
@thomas-maschler done